public/components/trace_analytics/components/dashboard/__tests__/__snapshots__/dashboard.test.tsx.snap

```diff
@@ -305,7 +327,29 @@ exports[`Dashboard component renders dashboard 1`] = `
         "setAppTitle": [MockFunction],
         "setBadge": [MockFunction],
         "setBrand": [MockFunction],
-        "setBreadcrumbs": [MockFunction],
+        "setBreadcrumbs": [MockFunction] {
+          "calls": Array [
+            Array [
+              Array [
+                undefined,
+                Object {
+                  "href": "#/trace_analytics/home",
```

Comment:
is `trace_analytics` in url works? I thought it was removed in earlier pr

Reply:
that is probably from test files https://github.com/opensearch-project/dashboards-observability/blob/544c2f50521501c87deacf3136b0596686467821/public/components/trace_analytics/components/dashboard/__tests__/dashboard.test.tsx#L39

The unit test doesn't care what it should be in reality, as long as dummy arguments are all valid it serves its purpose